src/app/controllers/MachineController.ts

```diff
@@ -205,67 +205,12 @@ export default class MachineController {
 		} as MachineChartData
 	}
 
-	public getFallbackConfigurations(data: ProcessedStats): FallbackConfigurations {
```

Comment:
Removing the entire fallback handling made both `sync_eth2_fallback_configured` and `sync_eth2_fallback_connected` in `interface ProcessBase` obsolete. They should be removed.